### PR TITLE
Adds customization id query string

### PR DESCRIPTION
**Pull request by DevChima**
We're currently doing this redirect:
http://symptomcheck.co.za/?whatsappid=12345

But ADA now wants this: 
http://symptomcheck.co.za/?whatsappid=12345?customizationId=kh93qnNLps

Checked with ADA and url should be
http://symptomcheck.co.za/?whatsappid=12345&customizationId=kh93qnNLps

### CODE REVIEW

File: ada/views.py [hunk 1]
```diff
@@ -33,9 +33,11 @@ def clickActivity(request: HttpRequest, pk: int, whatsappid: str) -> HttpRespons
     else:
         store_url_entry = RedirectUrlsEntry(symptom_check_url=redirect_url)
         store_url_entry.save()
+        customizationId = "kh93qnNLps"
```
Comment (erikh360) [inline ada/views.py]: Can we add this to settings with this value as a default please?

**Review by erikh360 — COMMENTED** [review]
Just the one comment and I think a test would be nice for this.

File: ada/views.py (hunk 1 above)
Comment (DevChima) [reply ada/views.py]: Alright.

File: ada/views.py [hunk 2]
```diff
@@ -33,9 +33,11 @@ def clickActivity(request: HttpRequest, pk: int, whatsappid: str) -> HttpRespons
...
         url = f"{redirect_url.symptom_check_url}"
-        qs = urlencode({"whatsappid": whatsappid})
-        destination_url = f"{url}?{qs}"
+        whatsapp_qs = urlencode({"whatsappid": whatsappid})
```
Comment (erikh360) [inline ada/views.py]: Is it not possible to urlencode this all in one step?

Comment (DevChima) [reply ada/views.py]: Might be. Let me see

**Pull request closed without merging**